src/services/SPService.ts

```diff
@@ -735,4 +736,55 @@ export default class SPService implements ISPService {
       return substringSearch ? fieldValue.indexOf(lowercasedFilterText) > -1 : fieldValue.startsWith(lowercasedFilterText);
     });
   }
+
+  /**
+   * Gets the collection of view for a selected list
+   */
+  public async getViews(listId?: string, _webAbsoluteUrl?:string, _orderBy?: orderBy, filter?: string): Promise<ISPViews> {
```

Comment:
why prefix with `_` **some** of the parameters?
Could you please use the same standard naming approach?

Reply:
Updated as suggested in the latest commit. 

Thanks,
Nishkalank